KIPs/kip-11.md

```diff
@@ -0,0 +1,126 @@
+
+| Criteria                                                                                                                                                           | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Demand for the token                                                                                                                                               | Recent trade volume of the token pair in the market can be considered as an indicator of token demand. Higher trade volume would mean more fees for the KyberDMM protocol and liquidity providers over time.                                                                                                                                                                                                                                                                                                                                                          |
+| Project reputation, quality, and security                                                                                                                          | Since Rainmaker will encourage more users to purchase the project tokens and adopt related products, it is ideal to collaborate with the best and most promising projects in DeFi.                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Rainmaker doesn't encourage the purchase of new project tokens. It encourages more liquidity contribution to the project tokens.